build.gradle

```diff
@@ -57,8 +57,8 @@ repositories {
     else {
         jcenter()
         mavenCentral()
-        maven { url "https://repository.apache.org/snapshots" }
         maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+        maven { url "https://repository.apache.org/snapshots" }
```

Comment:
It's ok, but please take care not to include unnecessary change in a pull request.